nucypher/blockchain/eth/actors.py

```diff
@@ -763,17 +763,53 @@ def _verify_active_ritual(self, decryption_request: ThresholdDecryptionRequest):
                 f"Node not part of ritual {decryption_request.ritual_id}",
             )
 
-    def _verify_ciphertext_authorization(
+    def _verify_encryption_authorization(
```

Comment:
It's interesting how product development has contributed to this naming evolution.